src/types/ar.ts

```diff
@@ -15,7 +15,7 @@ export class AR {
 
 	toString(): string {
 		BigNumber.config({ DECIMAL_PLACES: 12 });
-		const w = new BigNumber(this.winston.toString());
+		const w = new BigNumber(this.winston.toString(), 10);
```

Comment:
Why is that?

Reply:
It specifies the base number system to use when considering the config setting. It might be superfluous since tests were passing before.

Reply:
Got ya